### ISSUE TITLE
Import feature freezes Burp

### ISSUE THREAD

**Issue by Regala** [Regala/burp-scope-monitor#4]
The import function often makes Burp freeze (it unfreezes after a while). From experience this is probably related to the locks mechanism.

**Comment by abdilahrf** [comment]
Yes, I got same issue while try to import/export the endpoints would love to see this resolved
